README.md

```diff
@@ -8,6 +8,7 @@
 When you install PatternFly 4, the package includes:
 
 - a single file for the entire compiled library: `node_modules/@patternfly/patternfly/patternfly.css`
+- a single file for the entire compiled library, with page based breakpoints: `node_modules/@patternfly/patternfly/patternfly-page-width-breakpoints.css`
```

Comment:
Out of scope, but I wonder if we should also include `patternfly-base.css` - that's required to use with the individual component CSS.

@srambach what do you think about "page width breakpoints" as a name? I think I referred to it as "full width" in my other comment.

Reply:
I think I like full width better than page width.

Reply:
@mattnolting wdyt about the name?